<launch>

	<node name="cameraTracking" pkg="linetracking" type="cameraTest" output="screen">
		<param name="video" value="/home/pgp/lineTracking_ros/src/linetracking/dataset/output.avi"/>
		<param name="vertical_thres" value="0.5"/>
		<param name="parallel_thres" value="2.0/3"/>
		<param name="scale"  value="1000.0"/>
		<param name="thetaX" value="0.1"/>
		<param name="thetaY" value="0.1"/>
		<param name="offset" value="0.45"/>
		<param name="parallel_bias" type="int" value="20"/>
		<param name="imgWidth" value="1280"/>
		<param name="imgHeight" value="720"/>
	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find linetracking)/rviz/camera.rviz"/>

</launch>
